packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -263,6 +263,55 @@ describe('<Tabs />', () => {
         expect(style.left).to.equal('60px');
         expect(style.width).to.equal('50px');
       });
+
+      it('should have "right" for RLT', function test() {
+        if (isJSDOM) {
+          this.skip();
+        }
```

Comment:
It doesn't seem that we need to skip jsdom, it passes on it.

```suggestion
      it('should have "right" for RLT', () => {
```

Reply:
I think it should, I got the error after removed.